atlas-fastq-provider-config.sh.default

```diff
@@ -1,15 +1,18 @@
 ENA_SSH_HOST='sra-login-1'
 ENA_SSH_ROOT_DIR='/nfs/era-pub/vol1/'
-ENA_PRIVATE_SSH_ROOT_DIR='/nfs/dcc_arrayexpress/run'   
+ENA_PRIVATE_SSH_ROOT_DIR='/nfs/dcc_arrayexpress/run' # This no longer exists
+ENA_S3_PROFILE=
+ENA_S3_URL='http://hl.fire.sdo.ebi.ac.uk'
+ENA_S3_ROOT_PATH='s3://era-public'
+ENA_PRIVATE_S3_ROOT_PATH='s3://era-private/run'   
 ENA_FTP_ROOT_PATH='ftp://ftp.sra.ebi.ac.uk/vol1'
-ENA_HTTP_ROOT_PATH='https://hx.fire.sdo.ebi.ac.uk/fire/public/era'
+ENA_HTTP_ROOT_PATH='https://hl.fire.sdo.ebi.ac.uk/fire/public/era'
```

Comment:
Please note current (and correct) one in develop is: 
```suggestion
ENA_HTTP_ROOT_PATH='https://hl.fire.sdo.ebi.ac.uk/era-public'
```

Reply:
Addressed in a4909ba2a43bdda49dd0d663fc01d152b9f8e6ba, thanks!